playground/frontend/integration_test/standalone_miscellaneous_ui_test.dart

```diff
@@ -0,0 +1,256 @@
+    AnalyticsEvent(
+      category: kFeedbackCategory,
+      action: kClickSendFeedbackEvent,
+      label: text,
+    ),
+  );
+
+  expect(find.feedbackDropdownContent(), findsNothing);
+
+  // not enjoying and close
+  await wt.tap(find.feedbackThumbDown());
+  await wt.pumpAndSettle();
+
+  expect(find.feedbackDropdownContent(), findsOneWidget);
+
+  await wt.tap(find.feedbackDropdownCancelButton());
+  await wt.pumpAndSettle();
+
+  expect(find.feedbackDropdownContent(), findsNothing);
+}
+
+Future<void> _checkDescription(WidgetTester wt) async {
+  expect(find.descriptionPopoverButton(), findsOneWidget);
+
+  await wt.tap(find.descriptionPopoverButton());
+  await wt.pumpAndSettle();
+
+  expect(find.descriptionPopover(), findsOneWidget);
+
+  final example = wt.findPlaygroundController().selectedExample!;
+
+  expectContains(find.descriptionPopover(), find.text(example.name));
+  expectContains(find.descriptionPopover(), find.text(example.description));
+
+  await wt.tap(find.descriptionPopoverButton());
+  await wt.pumpAndSettle();
+
+  expect(find.descriptionPopover(), findsNothing);
+}
+
+Future<void> _checkOutputPlacement(WidgetTester wt) async {
+  Offset getCodeAreaCenter() => wt.getCenter(find.codeTextAreaWrapper());
+  Offset getOutputCenter() => wt.getCenter(find.outputWidget());
+
+  expect(find.splitView(), findsOneWidget);
+
+  await wt.tap(find.byKey(const ValueKey(OutputPlacement.left)));
+  await wt.pumpAndSettle();
+  expect(getCodeAreaCenter().dx > getOutputCenter().dx, true);
+
+  await wt.tap(find.byKey(const ValueKey(OutputPlacement.right)));
+  await wt.pumpAndSettle();
+  expect(getCodeAreaCenter().dx < getOutputCenter().dx, true);
+
+  await wt.tap(find.byKey(const ValueKey(OutputPlacement.bottom)));
+  await wt.pumpAndSettle();
+  expect(getCodeAreaCenter().dy < getOutputCenter().dy, true);
+}
+
+Future<void> _checkResizeOutput(WidgetTester wt) async {
+  final dragHandleStartPosition = wt.getCenter(find.dragHandle());
+  Future<void> returnDragHandleToStartPosition() async {
+    final currentPosition = wt.getCenter(find.dragHandle());
+    final offset = dragHandleStartPosition - currentPosition;
+    await wt.drag(find.dragHandle(), offset);
+    await wt.pumpAndSettle();
+  }
+
+  await _checkDragVertically(wt);
+  await returnDragHandleToStartPosition();
+
+  await _checkExcessivelyDragVertically(wt);
+  await returnDragHandleToStartPosition();
+
+  await wt.tap(find.byKey(const ValueKey(OutputPlacement.left)));
+  await wt.pumpAndSettle();
+
+  await _checkDragHorizontally(wt);
+  await returnDragHandleToStartPosition();
+
+  await _checkExcessivelyDragHorizontally(wt);
+  await returnDragHandleToStartPosition();
+}
+
+Future<void> _checkShortcutsModal(WidgetTester wt) async {
+  expect(find.shortcutsModal(), findsNothing);
+
+  expect(find.moreActions(), findsOneWidget);
+
+  AppLocalizations appLocale =
+      AppLocalizations.of(wt.element(find.moreActions()))!;
+
+  await wt.tap(find.moreActions());
```

Comment:
If you tap it, no need to check that it finds one widget.
Please check if there are other redundant expectations like this one.

Reply:
removed